HW_2/Task03.py

```diff
@@ -0,0 +1,47 @@
+# Напишите программу, которая принимает две строки вида “a/b” - дробь с числителем и знаменателем. 
+# Программа должна возвращать сумму и произведение* дробей. Для проверки своего кода используйте модуль fractions.gi
+
+from fractions import Fraction
+
+expression1 = (input('\nВведите первую дробь в формате 4/5: '))
+expression2 = (input('\nВведите вторую дробь в формате 4/5: '))
+
+lst1 = expression1.split("/")
+lst2 = expression2.split("/")
+a1 = int(lst1[0])
+a2 = int(lst1[1])
+b1 = int(lst2[0])
+b2 = int(lst2[1])
```

Comment:
Смотрите, как компактно, быстро и красиво можно это записать.
```suggestion
a1, a2 = map(int, expression1.split("/"))
b1, b2 = map(int, expression2.split("/"))
```